agent/run.sh

```diff
@@ -29,6 +29,11 @@ export CATTLE_HOME=${CATTLE_HOME:-/var/lib/cattle}
 
 check_debug
 
+RUNNING_IMAGE="$(docker inspect $(hostname)| jq -r .[].Config.Image)"
```

Comment:
Can you change to `docker inspect -f '{{.Config.Image}}' $(hostname)` with no `jq`